### PR TITLE
Support of sending soapAction in v1.2

### PR DESCRIPTION
**Pull request by vineet-jain**
https://github.com/strongloop/strong-soap/issues/170

### CODE REVIEW

**Comment by dhmlau** [comment]
@vineet-jain , thanks for your contribution.  Could you please fix the linting error of the commit message? 
```
First line should be 50 characters or less (saw 76)
```
See [this guideline](https://loopback.io/doc/en/contrib/git-commit-messages.html) for more details.

**Comment by vineet-jain** [comment]
@dhmlau - linting error of the commit message is resolved now

File: src/parser/wsdl/operation.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ class Operation extends WSDLElement {
       case 'operation': // soap:operation
         this.soapAction = child.$soapAction || '';
         this.style = child.$style || '';
+        this.soapActionRequired = child.$soapActionRequired || 'false';
```
Comment (raymondfeng) [inline src/parser/wsdl/operation.js]: Would it be better to convert the value to boolean?

Comment (vineet-jain) [reply src/parser/wsdl/operation.js]: Incorporated the comment to change this.soapActionRequired to boolean

**Comment by vineet-jain** [comment]
@raymondfeng - i have incorporated the review, can you relook at this PR?

**Comment by raymondfeng** [comment]
released as strong-soap@1.13.2

**Pull request merged**